pkg/skaffold/initializer/build/cli.go

```diff
@@ -48,17 +52,55 @@ func (c *cliBuildInitializer) ProcessImages(images []string) error {
 }
 
 func (c *cliBuildInitializer) BuildConfig() (latestV1.BuildConfig, []*latestV1.PortForwardResource) {
+	pf := []*latestV1.PortForwardResource{}
+
+	for _, manifestInfo := range c.manifests {
+		// Port value is set to 0 if user decides to not port forward service
+		if manifestInfo.Port != 0 {
+			pf = append(pf, &latestV1.PortForwardResource{
+				Type: "service",
+				Name: manifestInfo.Name,
+				Port: util.FromInt(manifestInfo.Port),
```

Comment:
what happens if two manifests specify the same local port? should we warn the user in that case?

Reply:
That's a good point, I think that we should probably do this in our interactive version, but with the `--artifact` flag, it seems that this should be handled on the end of whoever is utilizing the flag.

I can open an issue to modify the interactive version

Reply:
Only containers *within* a pod need to have distinct ports — pods have distinct port spaces.